### DISCUSSION TITLE
Guess Who's Back? Exodus Scam BitCoin Wallet Snap! - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#57] [General]
# Guess Who's Back? Exodus Scam BitCoin Wallet Snap! - Alan Pope's blog

Previously… Back in February, I blogged about a series of scam Bitcoin wallet apps that were published in the Canonical Snap store, including one which netted a scammer $490K of some poor rube’s coin.
The snap was eventually removed, and some threads were started over on the Snapcraft forum
Groundhog Day Nothing has changed it seems, because once again, ANOTHER TEN scam BitCoin wallet apps have been published in the Snap Store today.

https://popey.com/blog/2024/03/exodus-wallet-part-three/

**Comment by aranym0** [comment]
I've been reporting these and have noticed they usually take only a few hours, if that, to be reuploaded after a takedown. This is a bit ridiculous.

Why isn't Canonical starting a manual review period like last time when we know a bad actor is uploading malicious software?

**Reply by popey** [reply]
:man_shrugging:

**Comment by eson57** [comment]
And here we go again! This time it's appguard0x0000-
https://snapcraft.io/publisher/appguard0x0000
